frontend/src/pages/Home/index.tsx

```diff
@@ -42,10 +49,23 @@ const Home: React.FC = () => {
   }
   const { findMatch, loading } = match;
 
+  const isSmallerThan1100px = useMediaQuery('(max-width:1100px)');
+
   useEffect(() => {
     getQuestionCategories(dispatch);
   }, []);
 
+  useEffect(() => {
```

Comment:
how come need to listen to state.questions ahh? is it possible to just include all the functionality here into the button onclick function?

Reply:
ohh cuz if i immediately check state.questions after calling getQuestionList it doesn't work at times i think the state sometimes doesn't update immediately?

Reply:
yeah, I think because dispatch is like fire-and-forget so the state might not update in time

Reply:
ohh ok